packages/synapse-react-client/src/components/CardContainer/CardContainer.tsx

```diff
@@ -174,16 +188,24 @@ export function CardContainer(props: CardContainerProps) {
     )
   }
 
+  const isReleaseCardMediumList =
+    type === RELEASE_CARD && rest.releaseCardConfig?.cardSize === 'medium'
+
   return (
-    <div role="list">
-      {title && <h2 className="SRC-card-overview-title">{title}</h2>}
-      {!title && unitDescription && (
-        <TotalQueryResults frontText={'Displaying'} />
-      )}
-      {/* ReactCSSTransitionGroup adds css fade in property for cards that come into view */}
-      {cards}
+    <>
+      <Box
+        role="list"
+        sx={isReleaseCardMediumList ? releaseCardMediumListSx : defaultListSx}
```

Comment:
Add special styling for ReleaseCardMedium, so that text will be aligned across cards using a grid for the list of cards and a subgrid for each card's rows.

Reply:
Otherwise, the only other change here is to move the `showViewMoreButton` out of the list, so that the grid styling doesn't apply to the button.